front/app/containers/ProjectsShowPage/timeline/Timeline.tsx

```diff
@@ -483,13 +483,13 @@ const Timeline = ({
                 </Box>
               )}
             </RtlBox>
-            {!isBackoffice && selectedPhaseId && (
-              <PhaseDescription
-                projectId={projectId}
-                selectedPhaseId={selectedPhaseId}
-              />
-            )}
           </Phases>
+          {!isBackoffice && selectedPhaseId && (
+            <PhaseDescription
+              projectId={projectId}
+              selectedPhaseId={selectedPhaseId}
+            />
+          )}
```

Comment:
The `TabPanel` isn't supposed to appear nested under a `TabList`, so I've just moved this up one level so it appears as a sibling of `TabList` instead.

From what I'm reading, this appears to be the correct approach. The `Tab` within the `TabList` then references the `TabPanel` using an `aria-controls`:
![image](https://github.com/user-attachments/assets/6ad370e5-d955-433e-9b3f-eb91df449f2b)